build.proj

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <!-- required to build the projects in their specified order -->
+  <PropertyGroup>
+    <SerializeProjects>true</SerializeProjects>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <!-- To disable the restoration of packages, set RestoreDuringBuild=false or pass /p:RestoreDuringBuild=false.-->
+    <RestoreDuringBuild Condition="'$(RestoreDuringBuild)'==''">true</RestoreDuringBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Project Include="src/corehost/build.proj" />
+    <Project Include="pkg/dir.proj" />
+    <Project Include="src/dir.proj" />
+    <Project Include="test/dir.proj" Condition="'$(SkipTests)' != 'true'"/>
+    <Project Include="packaging/dir.proj"/>
+  </ItemGroup>
+
+  <Import Project="dir.targets" />
+
+  <Import Project="dir.traversal.targets" />
+  
+  <Import Project="$(ToolsDir)clean.targets" />
+
+  <PropertyGroup>
+    <TraversalBuildDependsOn>
+      BatchRestorePackages;
+      ValidateExactRestore;
+      CreateOrUpdateCurrentVersionFile;
+      BuildCustomTasks;
+      $(TraversalBuildDependsOn);
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="BuildCustomTasks">
+      <MSBuild Projects="tasks/core-setup.tasks.builds"/>
```

Comment:
I think a top-level directory named "tasks" is a poor name for this directory.  It feels like all our "build" specific stuff should be under a folder named "build".  So I can imagine this going under `build\tasks\`